packages/dva/dynamic.d.ts

```diff
@@ -1,2 +1,11 @@
-declare const dynamic: (resolve: (value?: PromiseLike<any>) => void) => void;
+import { DvaInstance } from 'dva';
+import React from 'react';
+
+interface Config {
+  app: DvaInstance,
+  models?: () => PromiseLike<any>[]
+  component: () => PromiseLike<any>
+}
```

Comment:
also `resolve?` and `LoadingComponent?` fields?

https://github.com/dvajs/dva/blob/master/packages/dva/src/dynamic.js#L60
https://github.com/dvajs/dva/blob/master/packages/dva/src/dynamic.js#L20

Reply:
already added